### ISSUE TITLE
linstor node set-property using equal syntax is working wrong

### ISSUE THREAD

**Issue by kvaps** [LINBIT/linstor-client#26]
```bash
# linstor n sp m15c34 --aux blabla=1       
SUCCESS:
Description:
    Node 'm15c34' modified.
Details:
    Node 'm15c34' UUID is: dcaa6ccb-b992-4dbb-b8da-a5e8904fb624
SUCCESS:
    (m15c18) Node changes applied.
SUCCESS:
    (m15c18) Resource 'one-vm-8471-disk-0' [DRBD] adjusted.
SUCCESS:
    (m10c28) Node changes applied.
SUCCESS:
    (m10c28) Resource 'one-vm-7912-disk-0' [DRBD] adjusted.
SUCCESS:
    (m10c14) Node changes applied.
SUCCESS:
    (m10c14) Resource 'one-vm-5395-disk-2' [DRBD] adjusted.
SUCCESS:
    (m10c42) Node changes applied.
SUCCESS:
    (m10c42) Resource 'one-vm-5642-disk-0' [DRBD] adjusted.
SUCCESS:
    (m10c15) Node changes applied.
SUCCESS:
    (m10c15) Resource 'one-vm-5713-disk-0' [DRBD] adjusted.
SUCCESS:
    (m10c20) Node changes applied.
SUCCESS:
    (m10c20) Resource 'one-vm-6177-disk-3' [DRBD] adjusted.
SUCCESS:
    (m10c24) Node changes applied.
SUCCESS:
    (m10c24) Resource 'one-vm-8452-disk-0' [DRBD] adjusted.
SUCCESS:
    (m10c36) Node changes applied.
SUCCESS:
    (m10c36) Resource 'one-vm-8460-disk-0' [DRBD] adjusted.
SUCCESS:
    (m10c40) Node changes applied.
SUCCESS:
    (m10c40) Resource 'one-vm-8513-disk-0' [DRBD] adjusted.
SUCCESS:
    (m14c13) Node changes applied.
SUCCESS:
    (m14c13) Resource 'one-vm-8513-disk-0' [DRBD] adjusted.
SUCCESS:
    (m14c6) Node changes applied.
SUCCESS:
    (m14c6) Resource 'one-vm-8479-disk-0' [DRBD] adjusted.
SUCCESS:
    (m13c28) Node changes applied.
SUCCESS:
    (m13c28) Resource 'one-vm-8460-disk-0' [DRBD] adjusted.
SUCCESS:
    (m10c19) Node changes applied.
SUCCESS:
    (m10c19) Resource 'one-vm-8452-disk-0' [DRBD] adjusted.
SUCCESS:
    (m10c7) Node changes applied.
SUCCESS:
    (m10c7) Resource 'one-vm-7912-disk-0' [DRBD] adjusted.
SUCCESS:
    (m10c21) Node changes applied.
SUCCESS:
    (m10c21) Resource 'one-vm-5642-disk-0' [DRBD] adjusted.
SUCCESS:
    (m10c27) Node changes applied.
SUCCESS:
    (m10c27) Resource 'one-vm-8460-disk-0' [DRBD] adjusted.
SUCCESS:
    (m10c27) Resource 'one-vm-8507-disk-2' [DRBD] adjusted.
SUCCESS:
    (m13c13) Node changes applied.
SUCCESS:
    (m13c13) Resource 'one-vm-5615-disk-0' [DRBD] adjusted.
SUCCESS:
    (m13c13) Resource 'one-vm-8507-disk-2' [DRBD] adjusted.
SUCCESS:
    (m14c43) Node changes applied.
SUCCESS:
    (m14c43) Resource 'one-vm-8131-disk-0' [DRBD] adjusted.
SUCCESS:
    (m14c12) Node changes applied.
SUCCESS:
    (m14c12) Resource 'one-vm-5507-disk-2' [DRBD] adjusted.
SUCCESS:
    (m10c30) Node changes applied.
SUCCESS:
    (m10c30) Resource 'one-vm-6177-disk-3' [DRBD] adjusted.
SUCCESS:
    (m14c34) Node changes applied.
SUCCESS:
    (m14c34) Resource 'one-vm-7010-disk-0' [DRBD] adjusted.
SUCCESS:
    (m13c33) Node changes applied.
SUCCESS:
    (m14c2) Node changes applied.
SUCCESS:
    (m13c33) Resource 'one-vm-5507-disk-2' [DRBD] adjusted.
SUCCESS:
    (m14c2) Resource 'one-vm-5615-disk-0' [DRBD] adjusted.
SUCCESS:
    (m10c22) Node changes applied.
SUCCESS:
    (m10c22) Resource 'one-vm-5713-disk-0' [DRBD] adjusted.
SUCCESS:
    (m10c26) Node changes applied.
SUCCESS:
    (m10c26) Resource 'one-vm-5395-disk-2' [DRBD] adjusted.
SUCCESS:
    (m14c25) Node changes applied.
SUCCESS:
    (m14c25) Resource 'one-vm-6287-disk-2' [DRBD] adjusted.
SUCCESS:
    (m15c33) Node changes applied.
SUCCESS:
    (m15c33) Resource 'one-vm-8479-disk-0' [DRBD] adjusted.
SUCCESS:
    (m13c3) Node changes applied.
SUCCESS:
    (m13c3) Resource 'one-vm-8131-disk-0' [DRBD] adjusted.
SUCCESS:
    (m13c20) Node changes applied.
SUCCESS:
    (m13c20) Resource 'one-vm-6287-disk-2' [DRBD] adjusted.
SUCCESS:
    (m13c41) Node changes applied.
SUCCESS:
    (m13c41) Resource 'one-vm-7010-disk-0' [DRBD] adjusted.
SUCCESS:
    (m10c39) Node changes applied.
SUCCESS:
    (m10c39) Resource 'one-vm-8479-disk-0' [DRBD] adjusted.
SUCCESS:
    (m10c4) Node changes applied.
SUCCESS:
    (m10c4) Resource 'one-vm-8471-disk-0' [DRBD] adjusted.
SUCCESS:
    (m10c4) Resource 'one-vm-8507-disk-2' [DRBD] adjusted.
SUCCESS:
    (m10c4) Resource 'one-vm-8513-disk-0' [DRBD] adjusted.
SUCCESS:
    (m15c34) Node changes applied.
SUCCESS:
    (m15c34) Resource 'one-vm-5395-disk-2' [DRBD] adjusted.
SUCCESS:
    (m15c34) Resource 'one-vm-5507-disk-2' [DRBD] adjusted.
SUCCESS:
    (m15c34) Resource 'one-vm-5615-disk-0' [DRBD] adjusted.
SUCCESS:
    (m15c34) Resource 'one-vm-5642-disk-0' [DRBD] adjusted.
SUCCESS:
    (m15c34) Resource 'one-vm-5713-disk-0' [DRBD] adjusted.
SUCCESS:
    (m15c34) Resource 'one-vm-6177-disk-3' [DRBD] adjusted.
SUCCESS:
    (m15c34) Resource 'one-vm-6287-disk-2' [DRBD] adjusted.
SUCCESS:
    (m15c34) Resource 'one-vm-7010-disk-0' [DRBD] adjusted.
SUCCESS:
    (m15c34) Resource 'one-vm-7912-disk-0' [DRBD] adjusted.
SUCCESS:
    (m15c34) Resource 'one-vm-8131-disk-0' [DRBD] adjusted.
SUCCESS:
    (m15c34) Resource 'one-vm-8452-disk-0' [DRBD] adjusted.
SUCCESS:
    (m15c34) Resource 'one-vm-8460-disk-0' [DRBD] adjusted.
SUCCESS:
    (m15c34) Resource 'one-vm-8471-disk-0' [DRBD] adjusted.
SUCCESS:
    (m15c34) Resource 'one-vm-8479-disk-0' [DRBD] adjusted.
SUCCESS:
    (m15c34) Resource 'one-vm-8507-disk-2' [DRBD] adjusted.
SUCCESS:
    (m15c34) Resource 'one-vm-8513-disk-0' [DRBD] adjusted.
```

```bash
# linstor n lp m15c34
+----------------------------+
| Key              | Value   |
|============================|
| Aux/blabla       | 1=      |
| Aux/moonshot     | 15      |
| Aux/opennebula-1 | true    |
| CurStltConnName  | default |
| NodeUname        | m15c34  |
+----------------------------+
```

```
# linstor --curl n sp m15c34 --aux blabla=1
curl -X PUT -H "Content-Type: application/json" -d '{"override_props": {"Aux/blabla": "1="}}' http://localhost:3370/v1/nodes/m15c34
```

**Comment by kvaps** [comment]
Yep, my mistake was with using `=`, I should use another argument for value instead. Anyway, I'm not sure that this behavior is expected.

**Comment by rp-** [comment]
will be fixed in the next client release `1.0.13`
